project4/index.html

```diff
@@ -0,0 +1,139 @@
+
+<body>
+    <div class="top">
+        <nav>
+            <ul class="topnav">
+                <li><a href="#home">HOME</a></li>
+                <li><a href="#schedule">SCHEDULE</a></li>
+                <li><a href="#speakers">SPEAKERS</a></li>
+            </ul>
+            <a class="registrationbutton" href="#register">REGISTER</a>
+        </nav>
+        <h1>Conference: Creating a World of Sustainable Graphic Design</h1>
+        <ul class="underheading">
+            <li>San Diego, CA, 117 Olang Dr.</li>
+            <li>December 12, 2019</li>
+            <!-- add speakers -->
+        </ul>
+    </div>
+    <div class="lowerpage">
+        <div class="textbody">
+            <p>Sustainable practices for graphic designers include a wide range of issues. When creating traditional
+                print materials the toxicity of ink and paper and the sheer quantity of paper produced need to be
+                considered. In addition to these factors there are other phases of the life cycle of products that need
+                to be examined. To really determine the sustainability or carbon footprint of a product, one needs to
+                follow it through its entire life cycle. Questions need to be raised about how much fuel is being used
+                for shipping, what the final end product is, how long the life cycle is, and how long before the product
+                ends up as waste.
+                In Green Graphic Design author Brian Dougherty asks graphic designers to start at the end of the process
+                instead of the beginning. Imagine the best possible destiny for your design and visualize the process of
+                every phase from the final destination of your product at the end of its life cycle back to the design
+                studio. Consider everything from the time of its ultimate disposal to its conception including
+                transportation, warehousing, production, and manufacturing that may prevent green solutions from being
+                implemented.
+                In Cradle to Cradle: Remaking the Way We Make Things authors William McDonough and Michael Braungart
+                make a similar case for how sustainable practices need to be implemented. They assert that it’s not
+                enough for us to “reduce, reuse, and recycle.” They explain how products need to be designed from the
+                outset so that after their lives they will provide nourishment for something new. McDonough and
+                Braungart feel that when designers employ the intelligence of natural systems—the effectiveness of
+                nutrient cycling, the abundance of the sun’s energy—they can create products and systems that allow
+                nature and commerce to fruitfully co-exist. Source for
+                text: http://www.ethicsingraphicdesign.org/morality/sustainability/</p>
+        </div>
+        <section id="schedule">
+            <h1>Schedule</h1>
```

Comment:
Your page already has an `h1` and, ideally, a page should only have _one_ `h1` since it's like the title of the page content.
```suggestion
            <h2>Schedule</h2>
```